### DISCUSSION TITLE
Thoughts on Utilizing Ferium to Auto-update Mods?

### DISCUSSION THREAD

**Discussion by zmweske** [itzg/docker-minecraft-server#2753] [Ideas]
This project (https://github.com/gorilla-devs/ferium) looks pretty cool and could be used to allow for automatic updates/upgrades of mod/server versions. Idk if the best way to do this is just to ship the utility with the container or make some simple env variables for updating on some time basis, but it might be neat.

**Comment by itzg** [comment]
Thanks for pointing that out but to be honest I'm not interested in adding another tool since I have already implemented a CLI tool

https://github.com/itzg/mc-image-helper

That is what I use in the image for all the features like Modrinth, CurseForge, some of the server downloads, and much more.

**Comment by zmweske** [comment]
Ok, based on what I can find it doesn't seem to be able to update mods unless it updates an entire modpack, is that correct? I could be wrong as the docs are different if you're looking at the generic mods doc page or modrinth/spiget pages. 

Either way love the tools you've made and thanks for all your effort!

**Comment by tomcrusade** [comment]
@zmweske , regarding ferium with itzg integration, i think i found out how to do it. This might not be what exactly you need, but, hopefully this implementation might sufficient https://github.com/tomcrusade/docker_fabric_vanilla
